src/lib/routes/admin-api/tag-type.ts

```diff
@@ -85,8 +96,12 @@ class TagTypeController extends Controller {
                 openApiService.validPath({
                     tags: ['Tags'],
                     operationId: 'validateTagType',
+                    summary: 'Validate a tag type',
+                    description:
+                        'Validate if the body of the request is a valid tag schema and if the tame given to the tag type already exists, returning a 409 status code if it does.',
```

Comment:
Small clarification suggestion:

```suggestion
                        'Validates whether if the body of the request is a valid tag and whether the a tag type with that name already exists or not. If a tag type with the same name exists, this operation will return a 409 status code.',
```